Dockerfile

```diff
@@ -13,23 +13,44 @@ WORKDIR /usr/src/app
 # A wildcard is used to ensure both package.json AND package-lock.json are copied
 # where available (npm@5+)
 COPY package*.json ./
+# Expose port 4000 inside the container to the outside world
+# so that http://localhost:3000 routes the network traffic to
+# the container
+EXPOSE 3000
```

Comment:
This differs from the [proposed DCS Dockerfile](https://github.com/CriminalInjuriesCompensationAuthority/data-capture-service/blob/d84a7c3bd23f286b37a437077a8e967812f7a28e/Dockerfile) in that it has a single `EXPOSE`. Is that correct?

Reply:
this is the same as the current [DockerFile ](https://github.com/CriminalInjuriesCompensationAuthority/cica-web/blob/d9e8d2a3a1be174ffb8d880aa16d1589d157ef71/Dockerfile#L31)